closed/openj9_version_info.h.in

```diff
@@ -37,5 +37,6 @@
 #define OPENJ9_TAG                "@OPENJ9_TAG@"
 #define J9JDK_EXT_VERSION         "@J9JDK_EXT_VERSION@"
 #define J9JDK_EXT_NAME            "@J9JDK_EXT_NAME@"
+#define JAVA_VENDOR               "@COMPANY_NAME@"
```

Comment:
Why is this not added like jdk8, jdk11?
```
#define JAVA_VENDOR_URL "@VENDOR_URL@"
```

Reply:
It's related to the OpenJ9 code that uses JAVA_VENDOR_URL, it's only used up to Java 11. https://github.com/eclipse/openj9/blob/master/runtime/vm/vmprops.c#L611-L639